docs/source/gpus/multigpu-partition-access.md

```diff
@@ -0,0 +1,94 @@
+# Access to Multi-GPU Partition
+The `multigpu` partition in the HPC cluster allows users for extensive parallel processing. This setup is ideal for applications that require significant computational power, such as deep learning, scientific simulations, and large-scale data analysis.
+
+## Applying for Multi GPU reservation
+
+### Step 1
+
+- Use the partition application form link in here: [HPC Partition Request](https://bit.ly/NURC-PartitionAccess)
+- Select under "Partition Type" - `Multigpu - Partition (Testing Access)`
+- Fill in the number of GPUs and the type of GPU you'd like to test on. Please also provide a short description of your expected testing workload.
```

Comment:
I would add a Note somewhere on this step. (The testing has to be representative of the work that you are planning on completing but it does not have to be the full complete runs to get the timing data).

We should also push for testing on v100s if the jobs will fit scaled down so we are not allocating a100s for multigpu testing. If the job scales on a v100, it should scale on an a100 as well.